content/api/suppression-list.apib

```diff
@@ -4,48 +4,84 @@ description: Manage your suppression list - a list of recipient email addresses
 
 # Group Suppression List
 
-A suppression list - or exclusion list, as it is sometimes called - stores a recipient's opt-out preferences. It is a list of recipient email addresses to which you do NOT want to send email. The Suppression List API is used to manage your exclusion list entries.  An entry indicates whether the recipient opted out of receiving one of the following:
+A suppression list - or exclusion list, as it is sometimes called - stores a recipient's opt-out preferences.
+It is a list of recipient email addresses to which you do NOT want to send email.
+Each entry indicates whether the recipient opted out of receiving one of the following:
 
-* Transactional messages
-* Non-transactional messages
+* Transactional messages - single recipient messages that are used operationally, e.g. to reset a password or confirm a purchase.
+* Non-transactional messages - used to run email campaigns where a list of recipients are targeted, e.g. advertising a sales event.
 
-Transactional messages are single recipient messages that are used operationally, e.g. to reset a password or confirm a purchase; while non-transactional messages are used to run email campaigns where a list of recipients are targeted, e.g. advertising a sales event.
 
-In addition to your suppression list, SparkPost maintains a global suppression list across all customers.
+<banner status="info">When setting up your SparkPost account, we <strong>strongly recommend</strong> you import any suppression list you have from any previous service to avoid incorrectly sending mail to unsubscribed/invalid recipients.</banner>
 
-## Recipient Database Maintenance
+## Recipient Maintenance
 
-<banner status="danger">When initially configuring your SparkPost account, we <strong>strongly recommend</strong> you import any suppression list you have from any previous service to avoid incorrectly sending mail to unsubscribed/invalid recipients.</banner>
+It's good practice to maintain your recipient lists by removing recipients based on the bounce, unsubscribe and spam complaint events provided by SparkPost.
+These events are available from [webhooks](/api/webhooks), [message events](/api/message-events) and your suppression list.
```

Comment:
The events aren't available from the suppression list. The suppressions are. I'm being nitpicky. 😬